fstab.yaml

```diff
@@ -1,2 +1,2 @@
-# mountpoints:
-#   /: <shared-folder-url>
\ No newline at end of file
+mountpoints:
+  /: https://drive.google.com/drive/u/0/folders/1MGzOt7ubUh3gu7zhZIPb7R7dyRzG371j
```

Comment:
You probably want to revert this one...

Reply:
@kptdobe I think this is intentional and part of the new first run experience @davidnuescheler had in mind: immediately show some content and guide the developer on what to do next.

Reply:
ah yes, this even the PR description :)